markdown/ch5.markdown

```diff
@@ -430,32 +430,27 @@ print('The CSR and NumPy arrays are equal: ',
       np.all(s2 == csr.A))
 ```
 
-The ability to store large, sparse matrices is incredibly powerful!
-The combination of sparsity and linear algebra abounds.  For example,
+The ability to store large, sparse matrices, and perform computations on them,
+is incredibly powerful!
+
+The combination of sparsity and linear algebra abounds. For example,
 one can think of the entire web as a large, sparse, $N \times N$ matrix.
 Each entry $X_{ij}$ indicates whether web page $i$ links to page $j$.
 By normalizing this matrix and solving for its dominant eigenvector,
 one obtains the so-called PageRank—one of the numbers Google uses to
 order your search results. (You can read more about this in the next chapter!)
 
-Now, consider studying the human brain, and represent it as a large $M
-\times M$ graph, where there are $M$ nodes (positions) in which you
+As another example, we can represent the human brain as a large $m \times m$
+graph, where there are $m$ nodes (positions) in which you
 measure activity using an MRI scanner.  After a while of measuring,
 correlations can be calculated and entered into a matrix $C_{ij}$.
-The matrix is thresholded (which makes it sparse and fills it with
-ones and zeros), representing an adjacency matrix.  The dominant
-eigenvector of this matrix is then calculated.  The sign of each entry
-in the (length $M$) eigenvector groups the nodes into two sub-groups
-(see Newman (2006), Modularity and community structure in networks,
-https://doi.org/10.1073/pnas.0601602103).  Rinse and repeat to
-form more and more and more sub-groups [^submod].  It turns out that
-these subgroups, or communities, tell us a lot about functional
-regions of the brain!
-
-[^submod]: One has to be careful when subdividing the network into
-           more than one group.  The graph under analysis remains the
-           original, *not* the previously split result.  A correct
-           approach is explained in detail in Newman (2006).
+Thresholding this matrix produces a sparse matrix of ones and zeros. 
+The eigenvector corresponding to the second-smallest eigenvalue of this matrix
+partitions the $m$ brain areas into subgroups, which, it turns out,
+correspond to functional regions of the brain [^Newman]!
```

Comment:
No, it doesn't remove sparsity but it removes simplicity. =) it no longer becomes easy enough to state the problem in a paragraph with no caveats. 

How would you phrase the functionality of the resulting modules? Again, if we couch the statements with too many caveats, it becomes pretty useless as an in-passing example.